### PR TITLE
Reword D2K Encyclopedia

### PR DESCRIPTION
**Pull request by test1232156**
Wording improvements

### CODE REVIEW

File: mods/d2k/languages/rules/en.ftl [hunk 1]
```diff
@@ -492,75 +491,75 @@ actor-missile-tank =
...
-   .encyclopedia = The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.
+   .encyclopedia = The Harkonnen Combat Tank is effective against most vehicles, less so against lightly armored ones.
 
-    The Harkonnen Combat Tank is stronger than its counterparts, but also slower.
+    It is stronger than its counterparts, but also slower.
 
 actor-combat-tank-o =
    .name = Ordos Combat Tank
-   .encyclopedia = The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.
+   .encyclopedia = The Ordoes Combat Tank is effective against most vehicles, less so against lightly armored ones.
```
Comment (Mailaender) [inline mods/d2k/languages/rules/en.ftl]:
```suggestion
   .encyclopedia = The Ordos Combat Tank is effective against most vehicles, less so against lightly armored ones.
```
Also, you generally seem to be not repeating the unit name in the first sentence, but you do it here.

File: mods/d2k/languages/rules/en.ftl [hunk 2]
```diff
@@ -492,75 +491,75 @@ actor-missile-tank =
...
-   .encyclopedia = The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.
+   .encyclopedia = It is effective against most vehicles but is less suited for lightly armored targets.
 
-    Atreides Combat Tanks are very resistant to bullet and heavy-explosives, but vulnerable to missiles and high-caliber guns.
+    It is very resistant to bullets and heavy explosives, but vulnerable to missiles and high-caliber guns.
 
 actor-combat-tank-h =
    .name = Harkonnen Combat Tank
-   .encyclopedia = The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.
+   .encyclopedia = It pis effective against most vehicles but is less suited for lightly armored targets.
```
Comment (Mailaender) [inline mods/d2k/languages/rules/en.ftl]:
```suggestion
   .encyclopedia = It is effective against most vehicles but is less suited for lightly armored targets.
```

Also, this sentence is duplicated. You can use https://projectfluent.org/fluent/guide/references.html instead.

Comment (PunkPun) [reply mods/d2k/languages/rules/en.ftl]: I disagree. Duplication is encouraged by fluent guidelines

Comment (Mailaender) [reply mods/d2k/languages/rules/en.ftl]: The first commit is solely about deduplication...

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/616861542806832c01ff5ca845b00ab7f743d5f3...a044e3c3ea22ea9ce23206e7c10acd728452f0e1)

**Pull request merged**